examples/c/CMakeLists.txt

```diff
@@ -9,5 +9,10 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-serialbox_add_c_example(example-01-laplacian-c example-01-laplacian.c)
+yoda_add_unittest(

```

Comment:
First example of how to build a unittest (i.e. binary) target. Yoda will build the binary and add a CMake test for that target. Most of the semantic here should be clear, but I will clarify here the difference between 
LIBRARIES -> specifies external library dependencies (i.e. they are not targets of our project). For example libpthread.so
DEPENDS -> specifies targets of our project (or imported projects) 

The difference is used here becaue all the targets on `DEPENDS` they carry properties. 
For example SerialboxCStatic might depend on other libraries that should be propagated when linking our executable (not in this case because we are using a static library, but it would happen if we would have chosen a shared library). 
In this particular example, serialbox-c depends on serialbox/core. So when compiling this target, our example will `#include` files of serialbox-c, which at the same time will `#include` headers from serialbox/core. 
At this point, if we dont include explicitly the include path of serialbox/core (and all the recursive dependencies) the compilation will failed. 
But how was it working before? Well, we used the CMake anti-pattern `include_directories`. 
How do we solve it now? Well, we dont have to do it, since yoda does it, it propagates all the include directories of the dependent targets. Isn't beautiful?
If you are curious to know how CMake resolves this, here is the magic expression to propagate dependent include directories:
https://github.com/cosunae/yoda/blob/create_library/cmake/yodaAddExecutable.cmake#L56